src/components/dialog/index.js

```diff
@@ -12,8 +12,8 @@ const Dialog = ({ children, goBack, onKeyPress }) => {
     }, []);
 
     function handleKeyPress(event) {
-        // case for 'enter'
-        if (event.keyCode === 13) {
+        // case for 'enter' and 'i' for closing inventory
```

Comment:
We might want to change this to explain what's going on here a little better.

Reply:
Would you say something along the lines of "check if a key is pressed and bound to an action" to be appropriate here?

Reply:
Yeah, something like that. Maybe let them know that it defaults to if enter has been pressed?

Reply:
Sorry, what do you mean by "it defaults to if enter has been pressed'?

Reply:
Ohh, you removed the 'event.keyCode === 13' part. We should probably have that as a fallback, otherwise we'd need to change every instance of Dialog which has an onKeyPress so that you can simply press enter to close it.

Reply:
Ah alright then

Reply:
Comments have been added, would you say that 3 lines of comments are too much though?

Reply:
I think removing the first line of the comment is probably appropriate, because it's sort of wrong now.

Reply:
And theres no need to specify the exact key pressed right? Or should I include that as well?

Reply:
I shouldn't think so.